src/modules/processing/matchActionDecider.js

```diff
@@ -95,7 +95,7 @@ export default function(r, host, userFormat, isAudioOnly, lang, isAudioMuted, di
             switch (host) {
                 case "videoclip":
                     if (r.urls.endsWith(".m3u8")) {
-                        params = { type: "render" }
+                        params = { type: "stream" }
```

Comment:
changing render to stream doesn't change anything, just forgot to change that line again